09-01-24/Destructuring-Arrays/js-modern-syntax_destructuring-arrays/index.js

```diff
@@ -6,41 +6,59 @@ console.clear();
 
 const item = ["Egg", 0.25, 12];
 
+export const [name, price, quantity] = item;
+console.log(name, price, quantity);
 // example: export const [value1, value2] = array;
 
 // EXERCISE 2
 // Use array destructuring to extract the variables "firstNameOfChris", "lastNameOfChris" and "ageOfChris".
 
 const personChris = [12, "Chris", "Owen"];
 
+export const [ageOfChris, firstNameOfChris, lastNameOfChris] = personChris;
+console.log(ageOfChris, firstNameOfChris, lastNameOfChris);
+
 // EXERCISE 3
 // Use array destructuring to extract the variables "firstNameOfAlex" and "lastNameOfAlex".
 // Make sure not to have unused variables.
 
 const personAlex = ["Alex", 12, "Main"];
 
+
+export const [firstNameOfAlex, ,lastNameOfAlex] = personAlex;
+console.log(firstNameOfAlex, lastNameOfAlex);
+
+
 // EXERCISE 4
 // Use array destructuring to extract the last name from the array
 // as variable called "lastName".
 
 const students = ["Christina", "Jon", "Alexandare"];
 
+export const [ , ,lastName] = students;
+console.log(lastName);
+
 // EXERCISE 5
 // Use array destructuring to extract all of the names from this nested array.
 // Assign the given order of students to variables called "student1" to "student5".
 
 const nestedStudents = ["Chris", ["Ahmad", "Antigoni"], ["Toby", "Sam"]];
 
+export const [student1, [student2,student3], [student4,student5]] = nestedStudents;
+console.log(student1,student2,student3,student4,student5);
+
+
 // EXERCISE 6
 // Spread the values of the values array into the function call of the add function.
 // Keep in mind that you need to export the variable 'result' to make the test work.
 
-const values = [1, 6, 7, 9, 12, 5, 4];
+const values = [1, 6, 7, 9, 12, 5, 4,100];
```

Comment:
Why did u add another number?
Btw, it is still working

Reply:
just for checking,

Reply:
I know that they did ask for it, more for my understanding